src/modules/blockchainApplication/connection/components/RequestSummary/index.js

```diff
@@ -8,8 +8,8 @@ import { extractAddressFromPublicKey } from '@wallet/utils/account';
 import { rejectLiskRequest } from '@libs/wcm/utils/requestHandlers';
 import { SIGNING_METHODS } from '@libs/wcm/constants/permissions';
 import { EVENTS } from '@libs/wcm/constants/lifeCycle';
-import { elementTxToDesktopTx } from '@transaction/utils/transaction';
-import { decodeTransaction } from '@transaction/utils/encoding';
+// import { elementTxToDesktopTx } from '@transaction/utils/transaction';
```

Comment:
this should be removed

Reply:
@eniolam1000752 I left them to understand what was it before, i don't know its working fine or not

Reply:
I will remove them now, from git blame we can know the previous change